tests/System.Buffers.Experimental.Tests/MemoryTests.cs

```diff
@@ -261,14 +251,19 @@ public override bool Release()
                 {
                     if (currentCount == 1)
                     {
-                        Dispose();
+                        Dispose(true);
```

Comment:
Prefer `((IDisposable)this).Dispose();` instead.

(The method does more than just delegate to `Dispose(true)`.)